redis-utils/src/main/java/org/ballerinalang/redis/BallerinaRedisDbErrors.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http:www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http:www.apache.orglicensesLICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.redis;
+
+/**
+ * Creates custom ballerina errors.
+ */
+public class BallerinaRedisDbErrors {
```

Comment:
Can't we move this to Constants.java class as we don't have any logic related to generate errors here?

Reply:
Updated in e26d6d2